charts/apisix/templates/configmap.yaml

```diff
@@ -101,9 +101,18 @@ data:
                                     # radixtree_uri_with_parameter: match route by uri with parameters
         ssl: 'radixtree_sni'        # radixtree_sni: match route by SNI(base on radixtree)
 
+      {{- $proxy_mode := "" }}
```

Comment:
Do you know what is the source of this change? 🤔

Reply:
https://github.com/apache/apisix/pull/9607